Resources/views/aside/folder.html.twig

```diff
@@ -0,0 +1,12 @@
+{% if treeChildItems|length > 0 %}
+    <ul>
+        {% for treeChildItem in treeChildItems %}
+             <li><a href="{{ path( 'ez_urlalias', { 'locationId': treeChildItem.contentInfo.mainLocationId } ) }}" title="{{ ez_content_name( treeChildItem.contentInfo ) }}">{{ ez_content_name( treeChildItem.contentInfo ) }}</a></li>
+        {% endfor %}
+    </ul>
+{% elseif isChild is not defined %}
```

Comment:
AFAICS this logic will render two levels when used through folder full view, as the sub-request from sub-navigation will enter here, and sub-request below will have `isChild` set to true? I kind of find it hard to read.